src/main.js

```diff
@@ -1,19 +1,128 @@
+//var Cover = require('../cover');
+// var covers = require('../data');
+// var titles = require('../data');
+// var descriptors = require('../data');
+
+var covers = [
```

Comment:
Copied from data.js file to this main.js file. Do we need to add a link from file to file, or is this okay?